Automattic/jetpack
# Jetpack Search: Add new marketplace extra field to postmeta_to_sync array

Fixes https://github.com/Automattic/wp-calypso/issues/69081

#### Changes proposed in this Pull Request:

* A new meta field that will include all the Marketplace fields serialized has been included in the list of fields to synchronize to WP.com

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* If possible, set up a local Jetpack environment with this branch applied
* Add a post with a custom field with the name `wpcom_marketplace_product_extra_fields` 
* Check that the field has been synchronized to WP.com
